pydicom/tests/test_dataset.py

```diff
@@ -164,7 +167,12 @@ def test_set_existing_data_element_by_name(self):
     def test_set_non_dicom(self):
         """Dataset: can set class instance property (non-dicom)."""
         ds = Dataset()
-        ds.SomeVariableName = 42
+        msg = (
+            r"Camel case attribute 'SomeVariableName' used which is not in "
+            r"the element keyword data dictionary"
+        )
+        with pytest.warns(UserWarning, match=msg):
+            ds.SomeVariableName = 42
```

Comment:
Not under a fixture, so is assuming the default value is "WARN".

Reply:
Done, I think. You just wanted a warning fixture added?

Reply:
Just thinking ahead to possibly changing the default to ERROR in _pydicom_ 3.x - so all tests should not assume the default value.